tests/test_main_notruncate.py

```diff
@@ -0,0 +1,52 @@
+# https://github.com/con2/emrichen/issues/54
+
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import yaml
+
+from emrichen.__main__ import main
+
+ORIGINAL_YAML = """
+a: 1
+b: 2
+"""
+
+UPDATE_YAML = """
+b: 3
+"""
+
+TEMPLATE_YAML = """
+!Merge
+  - !Include original.yaml
+  - !Include update.yaml
+"""
+
+
+def test_main_notruncate():
+    with TemporaryDirectory() as temp_dir:
+        temp_path = Path(temp_dir)
+        original_path = temp_path / "original.yaml"
+        update_path = temp_path / "update.yaml"
+        template_path = temp_path / "template.yaml"
+
+        for path, content in [
+            (original_path, ORIGINAL_YAML),
+            (update_path, UPDATE_YAML),
+            (template_path, TEMPLATE_YAML),
+        ]:
+            with open(path, 'w', encoding="UTF-8") as output_file:
+                output_file.write(content)
```

Comment:
Use [the tmpdir](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture), Luke!

```suggestion
def test_main_notruncate(tmpdir):
    original_path = tmpdir.join("original.yaml")
    original_path.write(ORIGINAL_YAML)
    update_path = tmpdir.join("update.yaml")
    update_path.write(UPDATE_YAML)
    template_path = tmpdir.join("template.yaml")
    template_path.write(TEMPLATE_YAML)
```